lib/.eslintrc.json

```diff
@@ -17,6 +17,18 @@
     "ecmaVersion": 2020
   },
   "rules": {
-    "node/no-unsupported-features/es-syntax": 0
+    "node/no-unsupported-features/es-syntax": 0,
+    "@typescript-eslint/no-unused-vars": [
+      "error", 
+      { 
+        "argsIgnorePattern": "^_"
```

Comment:
used to indicate parameter in the superclass method: https://github.com/okta/okta-auth-js/pull/1026/files#diff-bec99135b1c85591f00af9e8e5340f608875e800fea3d58517bb482aa60ef672R117